coalib/parsing/DefaultArgParser.py

```diff
@@ -119,6 +120,26 @@ def default_arg_parser(formatter_class=None):
                                 metavar='BOOL',
                                 help='Write the logs as json to a file '
                                 'where filename is specified as argument.')
+    if parser_type == 'coala-html':
```

Comment:
Now this is confusing. If the coala-html is a python package of its own ... why is the argparser for the coala-html package here ?
This wouldnt even be called right ? We probably want to add these in coala-html ?